packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -618,6 +627,25 @@ class StackViewport extends Viewport implements IStackViewport {
     return imagePlaneModule;
   }
 
+  /**
+   * Sets the default properties for the viewport
+   * @param ViewportProperties
+   * @param imageIdIndex If given, we st the default properties only for this image index
+   */
+  public setDefaultProperties(
+    ViewportProperties: StackViewportProperties = {},
+    imageIdIndex?: number
+  ): void {
+    if (!imageIdIndex) {
+      this.globalDefaultProperties = ViewportProperties;
+    } else {
+      this.defaultPropertiesByImageIdIndex.set(
```

Comment:
I would call this `perImageIdIndexDeafultProperties`, also now that i'm thinking by not based on imageId?